static/app/components/stream/group.tsx

```diff
@@ -95,8 +93,7 @@ function BaseGroupRow({
   narrowGroups = false,
   showLastTriggered = false,
 }: Props) {
-  const groups = useLegacyStore(GroupStore);
-  const group = groups.find(item => item.id === id) as Group;
```

Comment:
All callers already resolve the group via GroupStore.get(), they just weren't passing it down to the react component